tests/interpretation/test_rankings.py

```diff
@@ -0,0 +1,45 @@
+import random 
+import unittest 
+
+import neurox.interpretation.probeless as probeless 
+import neurox.interpretation.iou_probe as iou_probe
+import neurox.interpretation.gaussian_probe as guassian_probe
+import neurox.interpretation.linear_probe as linear_probe
+import numpy as np
+import torch 
+
+seed_value = 99
```

Comment:
Instead of defining globals like this, we should use `fixtures` (https://docs.pytest.org/en/stable/explanation/fixtures.html) that setup a consistent env for all tests using the fixtures; this is where we can also define the actual random data so its the same across all methods